### PR TITLE
Refresh visible data

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?226125#1139553

@orangejenny 

cc: @gcapalbo

### CODE REVIEW

File: tests/databrowser.js [hunk 1]
```diff
@@ -307,25 +309,55 @@ define([
...
+                                widget.input.promise.then(done);
+                            }
+                        }
+                    });
+                });
+
+                it("should not error for known properties", function() {
+                    assert.strictEqual(widget.getValue(), escapedDobProp);
+                    assert.lengthOf(widget.getControl().find('.label-datanode-unknown'), 0);
+                    event.fire("nodeError");
```
Comment (orangejenny) [inline tests/databrowser.js]: What are the two asserts after this verifying?

Comment (emord) [inline tests/databrowser.js]: Just that nothing has changed after data browser/external properties have been loaded. Probably not needed, but doesn't hurt either

Comment (orangejenny) [inline tests/databrowser.js]: picky, and i see you didn't invent this in this PR, but `nodeError` is a weird name if the purpose of the event is to force the external properties to load

Comment (emord) [inline tests/databrowser.js]: huh. I agree with you. I have no recollection of inventing that, but git doesn't lie https://github.com/dimagi/Vellum/commit/5b40c4c993df98588946a986af8996326b9ff97a will change it

**Pull request merged**